Pod/Classes/SeafRepos.m

```diff
@@ -407,4 +407,21 @@ - (BOOL)editable
 {
     return NO;
 }
+
+- (void)createLibrary:(NSString *)newLibName block:(void(^)(bool success, id repoInfo))completeBlock {
+    NSString *requestUrl = [NSString stringWithFormat:API_URL"/repos/"];
+    NSString *form = [NSString stringWithFormat:@"name=%@&desc=%@",newLibName,@"new library"];
+    [connection sendPost:requestUrl form:form
+                 success:
+     ^(NSURLRequest *request, NSHTTPURLResponse *response, id JSON) {
+         Debug("resp=%ld\n", (long)response.statusCode);
+         completeBlock(true,JSON);
+     }
+                 failure:
+     ^(NSURLRequest *request, NSHTTPURLResponse *response, id JSON, NSError *error) {
+         Warning("resp=%ld\n", (long)response.statusCode);
+         [self.delegate download:self failed:error];
```

Comment:
why  [self.delegate download:self failed:error] only in failure block, not in success block.